app/tests/lib/db_testing.py

```diff
@@ -18,10 +18,6 @@ def create_isolated_db(monkeypatch) -> None:
     """
     schema_name = f"test_schema_{uuid.uuid4().int}"
     monkeypatch.setenv("DB_SCHEMA", schema_name)
-    monkeypatch.setenv("POSTGRES_DB", "main-db")
-    monkeypatch.setenv("POSTGRES_USER", "local_db_user")
-    monkeypatch.setenv("POSTGRES_PASSWORD", "secret123")
-    monkeypatch.setenv("ENVIRONMENT", "local")
```

Comment:
No issue with removing these, but I've seen developers wanting to test things locally with environment+DB values set to something that is not `local` before, and forgetting to undo that before running tests (i've done it at least once).

Do we want to add any sort of assertion that you aren't trying to run test with `environment` or the DB set to something they shouldn't be?

Reply:
oh interesting, that's a good concern.
this makes me think we should have a test.env separate from local.env, similar to what we were talking about over slack. it'll also let us set LOG_LEVEL to "warning" or "critical" for test.env, which would alleviate the issue with noisy logs during test runs. thoughts?

Reply:
@chouinar does Alvin's PR for handling dotenv vars solve our issues? would you be able to port the pattern over?

Reply:
I think I need to see that PR move further along to know for sure. Will let you know when it's settled

Reply:
@chouinar Just checking if there was a conclusion to share back here

Reply:
This was done in https://github.com/navapbc/template-application-flask/pull/188, so this PR is now obsolete. I'll close